scipy/optimize/_optimize.py

```diff
@@ -2248,12 +2248,19 @@ def get_bracket_info(self):
             if (xa > xc):  # swap so xa < xc can be assumed
                 xc, xa = xa, xc
             if not ((xa < xb) and (xb < xc)):
-                raise ValueError("Not a bracketing interval.")
+                 raise ValueError(
+                    "Bracketing values (xa, xb, xc) do not"
+                " fulfill this requirement: (xa < xb) and (xb < xc)"
+                )
             fa = func(*((xa,) + args))
             fb = func(*((xb,) + args))
             fc = func(*((xc,) + args))
             if not ((fb < fa) and (fb < fc)):
-                raise ValueError("Not a bracketing interval.")
+                 raise ValueError(
+                    "Bracketing values (xa, xb, xc) do not"
+             " fulfill this requirement: (f(xb) < f(xa)) and (f(xb) < f(xc))"
+                )
```

Comment:
The indentation is not correct. You have to fix it that way to respect the line length as well.
```suggestion
                 raise ValueError(
                    "Bracketing values (xa, xb, xc) do not fulfill"
                    " this requirement: (f(xb) < f(xa)) and (f(xb) < f(xc))"
                )
```